cal/cal/static/css/calendaranalytics.scss

```diff
@@ -25,6 +25,12 @@ button {
   padding: 0 5 5 5;
 }
 
+.textbox input {
```

Comment:
Reason is that we're using an extra selector when we don't need one. The styling is only ever needed for the inner element, so we don't need to select the outer element and then reselect the inner element.